### DISCUSSION TITLE
Aggregating several ArchR projects

### DISCUSSION THREAD

**Discussion by nonafarbehi2018** [GreenleafLab/ArchR#960] [Questions / Documentation]
Hi, dear ArchR developers,

I am trying to deal with my massive 10X multiome dataset and I am keen to use ArchR to aggregate the scATAC data and perform downstream analysis. My first question is do you have any guidelines about applying ArchR package to multiome data?
and my second question is how I can aggregate various ArchR projects? I have done the initial analysis of each dataset separately using ArchR and saved the arrow files as well as projects.rds files.

Thanks
Nona

**Comment by rcorces** [comment]
Please note that this is not a bug and the Issue submission system clearly states that Issues are only for bugs / errors and Discussions are for questions/feature requests. I'm transferring your Issue to a Discussion.

**Comment by rcorces** [comment]
You can find a link the the Multiome tutorial on the ArchR github page - https://github.com/GreenleafLab/ArchR

You cannot aggregate ArchR projects. You can only aggregate arrows into a single ArchR project.
